docs/howto/recalculate_datasets.md

```diff
@@ -0,0 +1,19 @@
+# How to recalculate a Dataset
+
+ Recalculating a dataset is necessary when any of the following tasks are completed in Horreum
+
+* Created a schema
+* Updated a schema (or sub-schema)
+* Changed the number of Transformer variables
```

Comment:
There's no thing such as 'Transformer variables'. I would rather say that if you changed a Transformer, or changed the Transformers selection in Test we don't apply that automatically to previously uploaded Runs, and therefore you need to trigger the process manually.

Reply:
Removed mention of variables. Added Transformer selection modification.